corporate/lib/stripe_event_handler.py

```diff
@@ -102,6 +102,7 @@ def handle_checkout_session_completed_event(
         billing_session.update_or_create_stripe_customer(payment_method)
         process_initial_upgrade(
             user,
+            CustomerPlan.STANDARD,
```

Comment:
For both of these, ideally we'll get the information from the metadata. But that requires updating the generated stripe fixtures and that is a separate rabbit hole for now.

Making these hard-coded here should make it obvious that a follow-up for all the uses of a hard-coded CustomerPlan tier values in the billing upgrade path. But maybe worth a temporary comment as well?

Reply:
I think a temporary comment is probably unnecessary; we are going to audit `git grep CustomerPlan.STANDARD` at least one more time afte rthis.